### PR TITLE
TIQR-215: demo works with package now

### PR DESCRIPTION
**Pull request by bvboldrik**

### CODE REVIEW

File: Sources/TiqrCoreObjC/Classes/AboutViewController.m [hunk 1]
```diff
@@ -63,7 +62,7 @@ - (void)viewDidLoad {
     [self.okButton setTitle:NSLocalizedStringFromTableInBundle(@"ok_button", nil, SWIFTPM_MODULE_BUNDLE, @"OK") forState:UIControlStateNormal];
     self.okButton.layer.cornerRadius = 5;
 
-    NSString *version = [TiqrConfig valueForKey:@"CFBundleShortVersionString"];
+    NSString *version = [TiqrConfig valueForString:@"CFBundleShortVersionString"];
```
Comment (johankool) [inline Sources/TiqrCoreObjC/Classes/AboutViewController.m]: valueForString seems like a worse name that valueForKey to me.

File: Sources/TiqrCoreObjC/Classes/TiqrCoreManager.m [hunk 2]
```diff
@@ -68,41 +92,28 @@ - (void)startWithOptions:(NSDictionary *)launchOptions {
...
-        [application registerUserNotificationSettings:settings];
-        
-	}
-    #endif
-
-    return;
-}
+        if (info != nil) {
+            [self startChallenge:[info valueForKey:@"challenge"]];
+            return;
```
Comment (johankool) [inline Sources/TiqrCoreObjC/Classes/TiqrCoreManager.m]: return at the end of a method seems useless.

File: Sources/TiqrCoreObjC/Resources/General/Config.plist [hunk 3]
```diff
@@ -26,7 +26,7 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>$(MARKETING_VERSION)</string>
+	<string>1.0</string>
```
Comment (johankool) [inline Sources/TiqrCoreObjC/Resources/General/Config.plist:29]: This seems unintentional.

Comment (bvboldrik) [reply Sources/TiqrCoreObjC/Resources/General/Config.plist:29]: this is the package version, $(MARKETING_VERSION) does not work

**Review by johankool — APPROVED** [review]

**Pull request merged**